themes/socialbase/assets/css/admin-toolbar.css

```diff
@@ -1,35 +1,41 @@
 .toolbar-vertical #block-filter:target {
-  transform: translateY(39px) translateX(0);
+  -webkit-transform: translateY(39px) translateX(0);
```

Comment:
I don't think `-webkit-` prefixed version is required anymore.

Reply:
This issue is not relevant